pkg/vminetworkscontroller/vmi_controller_test.go

```diff
@@ -195,7 +195,7 @@ var _ = Describe("vmi IPAM controller", Serial, func() {
 					ObjectMeta: metav1.ObjectMeta{
 						Name:      "vm1.randomnet",
 						Namespace: "ns1",
-						Labels:    ownedByVMLabel(vmName),
+						Labels:    ownedByVMLabel("ns1", vmName),
```

Comment:
can't we use the constant here ?

Reply:
since 2 lines above it is hardcoded i followed the same semantics